### PR TITLE
KAA-1603: [Reviewed] Using Kaa C SDK on Linux. [master]

### PR DESCRIPTION
**Pull request by hunch7**

### CODE REVIEW

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md [hunk 1]
```diff
@@ -4,145 +4,147 @@ title: Linux
...
+{:.note}
 
- - **Host OS:** Ubuntu 14.04 LTS Desktop 64-bit.
+## Prerequisites
 
-### Dependencies
+To build Kaa C SDK, make sure to install the following components:
 
-To build C SDK make sure that the following components are installed:
+ - CMake (minimum required version is 3.5.2).
```
Comment (Polinaa) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md]: The default cmake version on Ubuntu 14.04 is 2.8.12.

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md [hunk 2]
```diff
@@ -4,145 +4,147 @@ title: Linux
...
 
         touch CMakeLists.txt
 
-    Add the following code to top-level `CMakeLists.txt`.
-    The first line sets the minimum cmake version required to build kaa-application project. The second line sets project name and the language of the project:
+    Add the following code to the `CMakeLists.txt` file.
+    The first line sets the minimum CMake version required to build a `kaa-application` project.
+    The second line sets project name and language.
 
         cmake_minimum_required(VERSION 3.5.2)
```
Comment (Polinaa) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md]: Change version to 2.8.12.

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md [hunk 3]
```diff
@@ -4,145 +4,147 @@ title: Linux
...
-    Compile executable named `kaa-app` using `src/kaa-application.c`, and link it with `kaac`.
+    Compile the executable file `kaa-app` using `src/kaa-application.c`, and link it with `kaac`.
 
         add_executable(kaa-app src/kaa-application.c)
         target_link_libraries(kaa-app kaac)
 
-    Full `CMakeLists.txt` code:
+    See the full `CMakeLists.txt` code below.
 
         cmake_minimum_required(VERSION 3.5.2)
```
Comment (Polinaa) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md]: Change version to 2.8.12.

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md (hunk 1 above)
Comment (hunch7) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md]: Done.

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md (hunk 2 above)
Comment (hunch7) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md]: Done.

Comment (hunch7) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md]: Done.

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md (hunk 3 above)
Comment (hunch7) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md]: Done.

**Review by Polinaa — APPROVED** [review]

**Pull request merged**